### PR TITLE
Add Multiple Target Support

### PR DESCRIPTION
**Pull request by YogaSakti**
it's done UwU 🥴

### CODE REVIEW

File: downloader.js [hunk 1]
```diff
@@ -1,21 +1,18 @@
 const {loadData, storeData, downloadImage, upsertDirectory, downloadVideo} = require('./util')
-
-
 const story = loadData("metadata-story.json")
-
 const storyDownloaded = story.map(content => {
     if(!content.downloaded){
         if(content.mediaType == 1){ // image
             const dir = `./images/${content.path}`
```
Comment (2pai) [inline downloader.js]: can we make the path is group by username?
i.e. `./images/${content.username}/${content.path}` for more clarity. 
then, we didn't have to change the filename.

**Review by 2pai — CHANGES_REQUESTED** [review]
Good works, maybe need a few improvements.

File: downloader.js (hunk 1 above)
Comment (YogaSakti) [reply downloader.js]: okay

File: dispatcher.js [hunk 2]
```diff
@@ -17,15 +13,15 @@ metadataStory
         if (story.mediaType == 1) { // Image
             bot.sendPhoto(chatID, story.url)
                 .then((data) => {
-                    console.log("Success send " + story.id + " to " + data.chat.title + " (" + data.chat.id + ")")
+                    console.log("Success send " + story.id + " to " + data.chat.title || data.chat.username || data.chat.first_name || data.chat.last_name + " (" + data.chat.id + ")")
```
Comment (2pai) [inline dispatcher.js]: can we use `Success send  ${story.id} to ${data.chat.title || data.chat.username || data.chat.first_name || data.chat.last_name } ${data.chat.id  )` instead ?

File: dispatcher.js [hunk 3]
```diff
@@ -17,15 +13,15 @@ metadataStory
...
                     updateData("./metadata-story.json", story.id, true)
                 }).catch((error) => {
                     console.log("failed to send: " + story.id)
                 })
         } else if (story.mediaType == 2) {  // Videos
             bot.sendVideo(chatID, story.url)
                 .then((data) => {
-                    console.log("Success send " + story.id + " to " + data.chat.title + " (" + data.chat.id + ")")
+                    console.log("Success send " + story.id + " to " + data.chat.title || data.chat.username || data.chat.first_name || data.chat.last_name + " (" + data.chat.id + ")")
```
Comment (2pai) [inline dispatcher.js]: idem

**Comment by 2pai** [comment]
cc @YogaSakti

**Review by 2pai — APPROVED** [review]
LGTM

**Pull request merged**